### PR TITLE
Refractor frosh routes

### PR DESCRIPTION
**Pull request by lukewarmtemp**
#23

### CODE REVIEW

**Review by Cali0707 — APPROVED** [review]
Looks great!

**Review by NeoLou — APPROVED** [review]
Looks good!

File: server/src/controllers/FroshController.js [hunk 1]
```diff
@@ -19,6 +21,50 @@ const FroshController = {
...
+// 				console.log("new password saved");
+// 				res.status(200).send({
+// 					message: 'Password Updated'
+// 				});
+// 			});
+// 		});
+// 	} else {
+// 		res.status(404).json('No user exists in db to update');
+// 	}
+//   },
```
Comment (Cali0707) [inline server/src/controllers/FroshController.js:67]: Can you get rid of this commented-out code?

File: server/src/routes/froshRoutes.js [hunk 2]
```diff
@@ -16,5 +16,7 @@ const router = express.Router();
  *
  */
 router.post('/register', FroshController.registerFrosh);
+// router.get('/reset', FroshController.froshReset);
+// router.put('/updatePasswordViaEmail', FroshController.updateFroshPassViaEmail);
```
Comment (Cali0707) [inline server/src/routes/froshRoutes.js:20]: Get rid of commented-out code

File: server/src/server.js [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+
+const app = express();
+
+app.use(bodyParser.json());
+
+app.use('/frosh', froshRouter);
+
+app.get('*', (req, res) => {
+  res.status(200).send('Hello, Calum!!');
+});
```
Comment (Cali0707) [inline server/src/server.js:13]: Thanks for fixing the order here -- looks great!!

File: server/src/services/FroshServices.js [hunk 4]
```diff
@@ -56,6 +56,72 @@ const FroshServices = {
...
+  //   return (Frosh.findOne({
+  //     email: new RegExp(`^${froshEmail}$`, 'i')
+  //   }));
+  // },
+
+  // async resetPassTokenFindFrosh(token) {
+  //   return (Frosh.findOne({
+  //     resetPasswordToken: token,
+  //   }));
+  // },
```
Comment (Cali0707) [inline server/src/services/FroshServices.js:90]: Get rid of the commented-out code

File: server/src/services/FroshServices.js [hunk 5]
```diff
@@ -56,6 +56,72 @@ const FroshServices = {
...
+  
+  async initFroshGroups(groups) {
```
Comment (Cali0707) [inline server/src/services/FroshServices.js:92]: Please add a [jsdoc comment](https://jsdoc.app/about-getting-started.html) to document this function - all you need is a one sentence description, and then params and a return type

**Review by Cali0707 — CHANGES_REQUESTED** [review]
Just a few changes and this should be good to go -- great work!!

**Comment by Cali0707** [comment]
@lukewarmtemp any updates on the status of this?

**Pull request closed without merging**